src/redis/main/rapidcore.redis.csproj

```diff
@@ -21,7 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="StackExchange.Redis" Version="2.*" />
-    <PackageReference Include="StackExchange.Redis.Extensions.Core" Version="5.5.0" />
+    <PackageReference Include="StackExchange.Redis.Extensions.Core" Version="6.*" />
```

Comment:
**issue**: I am not crazy about the wildcard, as it means that builds made at different points in time (or just on different servers) can lead to different binaries.

Do they follow SemVer?

Reply:
Yes it seems like they do indeed follow SemVer in that package. But I'm open for pinning the version

Reply:
I see the point of the wildcard, I just also see possible issues with it stemming from unpredictability.

Reply:
using the 6.* should make it easier for consumers to lock in a specific version if I'm not mistaken

Reply:
Let's try the wildcard. Assuming they do not make a mistake and release a breaking change as less than a major, the wildcard should provide some freedom for library consumers to provide their own within the 6.x range.